packages/@okta/vuepress-theme-prose/components/ContentPage.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+  <Content />
+</template>
+
+<script>
+import { LAYOUT_CONSTANTS } from "../layouts/Layout";
+export default {
+  name: "ContentPage",
+  data() {
+    return {
+      anchors: [],
+      headingAnchorsMap: {},
+      paddedHeaderHeight: 0,
+    };
+  },
+  mounted() {
+    window.addEventListener("load", () => {
+      window.setTimeout(() => {
+        this.paddedHeaderHeight =
+          document.querySelector(".fixed-header").clientHeight +
+          LAYOUT_CONSTANTS.HEADER_TO_CONTENT_GAP;
+        this.scrollToActiveAnchor();
+        this.captureAnchors();
+      }, 500);
```

Comment:
what for using `500milsec` delay?

Reply:
I believe this could have been added originally as a safety net to win time until tree-nav(left sidebar) is rendered
going to [dev]test its removal on NF env